src/hinting.ts

```diff
@@ -131,6 +131,7 @@ class Hint {
         `
         modeState.hintHost.appendChild(this.flag)
         target.classList.add('TridactylHintElem')
+        target.classList.add(config.get("theme"))
```

Comment:
You're adding the theme class to the target element, which is the web content element that is getting hinted, not the hint label.

Check out the HintState destructor to see how hints are removed (Hint.hidden = true). You need to hook into that logic, maybe add a destructor to the Hint class, maybe just remove the theme when hidden = true.

Also, the theme shouldn't be a name likely to clash with web content CSS. Try "TridactylHintElemDark" or something.